static/js/modules/accordion.js

```diff
@@ -119,4 +121,4 @@ var accordion = {
   }
 };
 
-module.exports = accordion;
\ No newline at end of file
+module.exports = accordion;
```

Comment:
I'm a little confused about this one, but it looks like GitHub is displaying the diff wrong. I didn't delete that line, but it's showing up as deleted anyway. Check out the modified file at https://github.com/jmcarp/openFEC-web-app/blob/feature/browserify-ify/static/js/modules/accordion.js. It has that line intact.